### ISSUE TITLE
Possibility to raid from one live to another

### ISSUE THREAD

**Issue by mose** [JohnXLivingston/peertube-plugin-livechat#245]
This feature is one of the core community dynamic flow on Twitch. I put it here but actually it may belong to the Peertube project to implement this feature.

When a streamer prepares to end their stream, they should be proposed a list of currently live streamer, so they can send their community to them.

Obvisouly each streamer should be able to chose if they are open to be raided or not. Raidable streams may be taken from a short list of friended channels (see #244).

**Comment by dannekrose** [comment]
I thought about how I could simulate this somehow and I think this point is going to be pretty difficult to implement:

> Obvisouly each streamer should be able to chose if they are open to be raided or not. Raidable streams may be taken from a short list of friended channels (see https://github.com/JohnXLivingston/peertube-plugin-livechat/issues/244).

This will probably need to add some kind of flags on a per-PeerTube channel basis, but I'm not sure if it could be done purely with the livechat plugin.  I could see some issues also about trying to limit it to currently live streams, too, but I do also like the idea for the feature.  It would be nice to have for sure.

**Comment by JohnXLivingston** [comment]
> This will probably need to add some kind of flags on a per-PeerTube channel basis

Yes, and the v8.0.0 comes with a first "beta page" to customize chats per channel:

https://livingston.frama.io/peertube-plugin-livechat/documentation/user/streamers/channel/
https://livingston.frama.io/peertube-plugin-livechat/documentation/user/streamers/bot/

These are the first stones for many incoming features :)
